docs/compendium/substitute.md

```diff
@@ -30,6 +31,7 @@ input of request-body for substitution
 - `substfiles` - array of file substitutions specified by `original` and `xchange`
   - `original` - filename of the file from the base ERC
   - `xchange` - filename of the overlay ERC that will be exchanged for the original file
```

Comment:
In our discussions we use "base" vs. "overlay". here the properties are named "original" and "xchange".

Which variant do you prefer?